invenio_app_ils/records_relations/retriever.py

```diff
@@ -170,6 +170,11 @@ def get(self):
                 r = self._build_relation_obj(sibling_pid, name)
                 relations.setdefault(name, [])
                 relations[name].append(r)
+
+            # Pre-requisite: fields being sorted should be of the same type.
+            if sort_by and name in sort_by and name in relations.keys():
```

Comment:
Question: Does this mean that the `EDITION_RELATION` can only be sorted by `edition`, `LANGUAGE_RELATION` only by `language` and `OTHER_RELATION` by `other`?

Reply:
Yes!
Do we want the ability to sort by another field apart from the relation type?